README.md

```diff
@@ -30,21 +30,22 @@ The basic usage of this library is to submit blocking tasks to be executed by a
 
 require __DIR__ . '/../vendor/autoload.php';
 
+use Amp\Future;
 use Amp\Parallel\Worker;
-use Amp\Promise;
+use function Amp\launch;
 
 $urls = [
     'https://secure.php.net',
     'https://amphp.org',
     'https://github.com',
 ];
 
-$promises = [];
+$futures = [];
 foreach ($urls as $url) {
-    $promises[$url] = Worker\enqueueCallable('file_get_contents', $url);
+    $futures[$url] = launch(fn () => Worker\enqueueCallable('file_get_contents', $url));
```

Comment:
This isn't really related to this PR, but I guess we should rename the function, because it sounds like it returns a future. /cc @trowski